packages/chaire-lib-backend/src/models/db/default.db.queries.ts

```diff
@@ -194,8 +223,11 @@ const updateMultiple = async <T extends GenericAttributes, U>(
                     .transacting(trx);
             });
             const returningArray = await Promise.all(queries);
-            return returningArray;
-        })) as string[][];
+            return returningArray as string[][];
+        };
+        return await (options.transaction !== undefined
```

Comment:
Not required, but it could  be interesting to so if this function could be refactored to avoid an inner
function and an inline if like that. 
At least, please add some comments.